ilastik/plugins.py

```diff
@@ -161,17 +161,19 @@ def checkFilesExist(self, filename):
         ''' Check whether the files we want to export (when appending the base filename) are already present '''
         return False
 
-    def export(self, filename, hypothesesGraph, *, objectFeaturesSlot, labelImageSlot, rawImageSlot, bdvFilepathSlot):
+    def export(self, filename, hypothesesGraph, pluginExportContext):
         """Export the tracking solution stored in the hypotheses graph's "value" and "divisionValue"
         attributes (or the "lineageId" and "trackId" attribs). See https://github.com/chaubold/hytra for more details.
 
         :param filename: string of the file where to save the result (or folder where to put the export files)
         :param hypothesesGraph: hytra.core.hypothesesgraph.HypothesesGraph filled with a solution
-        :param objectFeaturesSlot (lazyflow.graph.InputSlot): connected to the RegionFeaturesAll
+        :param pluginExportContext (ilastik.plugins.PluginExportContext): instance of PluginExportContext containing data necessary for the export such as:
```

Comment:
sorry for the pedantry, but 155 characters....

Reply:
:white_check_mark: